sponsorship_compassion/views/invoice_line_view.xml

```diff
@@ -73,8 +73,37 @@
         </field>
     </record>
 
-    <!-- View children contained in invoice -->
     <record model="ir.ui.view" id="invoice_tree_compassion">
+        <field name="name">account.invoice.tree.compassion.ch</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id"
+               ref="base_transaction_id.invoice_tree_custom"/>

```

Comment:
This is not a good inheritance (it was in switzerland module where we inherit from switzerland modules but here we don't have access to them). Replace this inheritance to inherit the base tree view of account_invoice defined in the `account` module.